plugins/gadget.yaml

```
@@ -0,0 +1,42 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: gadget
+spec:
+  version: v0.1.0
+  homepage: https://github.com/kinvolk/inspektor-gadget
+  shortDescription: Collection of gadgets for debugging and introspecting apps
```

```suggestion
  shortDescription: Collection of gadgets for debugging and introspecting
```
apps is somewhat implicit.

And since this field is usually truncated at 50 chars, you can really make it fit by just  saying:
```suggestion
  shortDescription: Gadgets for debugging and introspecting
```